packages/server/graphql/public/mutations/autogroup.ts

```diff
@@ -0,0 +1,107 @@
+import getRethink from '../../../database/rethinkDriver'
+import Reflection from '../../../database/types/Reflection'
+import {getUserId, isTeamMember} from '../../../utils/authorization'
+import sendToSentry from '../../../utils/sendToSentry'
+import standardError from '../../../utils/standardError'
+import {GQLContext} from '../../graphql'
+import addReflectionToGroup from '../../mutations/helpers/updateReflectionLocation/addReflectionToGroup'
+import {MutationResolvers} from '../resolverTypes'
+
+type GroupedReflection = {
+  [groupTitle: string]: string[]
+}
+
+const autogroup: MutationResolvers['autogroup'] = async (
+  _source,
+  {meetingId}: {meetingId: string},
+  context: GQLContext
+) => {
+  const {authToken} = context
+  const viewerId = getUserId(authToken)
+  const r = await getRethink()
+  const meeting = await r.table('NewMeeting').get(meetingId).run()
+
+  if (!meeting) {
+    return standardError(new Error('Meeting not found'), {userId: viewerId})
+  }
+  if (meeting.meetingType !== 'retrospective') {
+    return standardError(new Error('Incorrect meeting type'), {userId: viewerId})
+  }
+  const {groupedReflectionsJSON, teamId} = meeting
+  if (!isTeamMember(authToken, teamId)) {
+    return standardError(new Error('Team not found'), {userId: viewerId})
```

Comment:
+1 I think we should return "Meeting not found" here so you cannot use this mutation to test for Meeting ids.

Reply:
Makes sense, I've updated the error message 👍